lib/kumascript/server.js

```diff
@@ -231,6 +235,52 @@ var Server = ks_utils.Class({
         res.json(data);
     },
 
+    /**
+     * A "liveness" endpoint for use by Kubernetes or other
+     * similar systems. A successful response from this endpoint
+     * simply proves that this Express app is up and running. It
+     * doesn't mean that its supporting services (like the macro
+     * loader and the document service) can be successfully used
+     * from this service.
+     */
+    liveness_GET: function (req, res) {
+        res.sendStatus(204);
+    },
+
+    /**
+     * A "readiness" endpoint for use by Kubernetes or other
+     * similar systems. A successful response from this endpoint goes
+     * a step further and means not only that this Express app is up
+     * and running, but also that one or more macros have been found
+     * and that the document service is ready.
+     */
+    readiness_GET: function (req, res) {
+        var msg = 'service unavailable ',
+            parts = url.parse(this.options.document_url_template),
```

Comment:
Good question. I think I like the idea of a separate config item, as that might be clearer and more controllable/flexible?